app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -133,8 +136,37 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
         setSchedule(scheduleData.events);
       }
     } catch (error) {
-      console.error('Could not load week data:', error);
       setError('Could not load week data.');
+      throw error;
```

Comment:
Are we doing `throw error`, or `throw new Error`? Because [here](https://github.com/LetsGetTechnical/gridiron-survivor/pull/530/files#diff-84b6ea6f4c76b2d0ee1408f1a109742c5d480061e971a02b3ffe177c312a090aR153) you use `throw new Error`

Reply:
@choir27 that is a great question.  Throw new error is used when we are customizing the error message.   You had asked me to change this to throw error since it show cover everything.

I think it is a valid point but we need @shashilo to weigh in on this.  It would cause a lot of code updates across lots of components to make this consistent.

Really good eye though for consistency!